app/controllers/build_commands_controller.rb

```diff
@@ -10,14 +10,10 @@ def show
 
   def update
     command = params[:command][:command]
-    if command.blank?
```

Comment:
so a user would not be able to unlink a command unless it's the last usage ... 
but we cannot allow editing a potentially reused/global command ... that might be bad ...
users could work around it by deleting it from other usage locations ...

how about: do the actual bugfix first ... make it not editable/destroyable if it's reused ... then we worry about usability later ...